src/CommunityToolkit.Maui.Core/CommunityToolkit.Maui.Core.csproj

```diff
@@ -56,4 +56,8 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net7.0-android'">
+    <PackageReference Include="Xamarin.Android.ShortcutBadger" Version="1.2.0" />
+  </ItemGroup>
```

Comment:
Let's avoid adding a NuGet dependency to `CommunityToolkit.Maui.Core`

We should also avoid taking a dependency on a non-Microsoft library. (Albeit, we did take a dependency on `Xam.Plugins.Android.ExoPlayer` for `CommunityToolkit.Maui.MediaElement`).

Perhaps you could reach out to this library owner to see if we could leverage his implementation for the toolkit?

Reply:
maybe reach out to alexrainman, as his does not just encapsulate another native library
https://github.com/alexrainman/Badge